editor/editor_properties_array_dict.h

```diff
@@ -81,6 +81,23 @@ class EditorPropertyDictionaryObject : public RefCounted {
 class EditorPropertyArray : public EditorProperty {
 	GDCLASS(EditorPropertyArray, EditorProperty);
 
+	struct Slot {
+		Ref<EditorPropertyArrayObject> object;
+		HBoxContainer *hbox;
+		int index = -1;
+		Variant::Type type = Variant::VARIANT_MAX;
+		bool as_ID;
```

Comment:
```suggestion
		bool as_id = false;
```
Use `snake_case` and also initialize values